keylime/tenant.py

```diff
@@ -1239,7 +1239,10 @@ def do_add_runtime_policy(self, args: Dict[str, str]) -> None:
         response = cv_client.post(
             f"/v{self.api_version}/allowlists/{self.runtime_policy_name}", data=body, timeout=self.request_timeout
         )
-        Tenant._jsonify_response(response)
+        response_json = Tenant._jsonify_response(response)
+
+        if response.status_code >= 400 and response.status_code < 500:
+            raise UserError(response_json)
```

Comment:
I am going to have to drop this since it causes issues with existing tests that we probably should resolve separately.

```
2023-03-17 16:23:45.310 - keylime.tenant - ERROR - {'code': 409, 'status': 'Runtime policy with name list2 already exists', 'results': {}}

:: [ 16:23:45 ] :: [   FAIL   ] :: Command 'keylime_tenant -c addruntimepolicy --runtime-policy policy.json --runtime-policy-name list2' (Expected 0, got 1)
```